experiment/bigquery/README.md

```diff
@@ -12,11 +12,18 @@ instructions](https://cloud.google.com/sdk/downloads#apt-get)).
 
 ## Scripts
 
-* `flakes.sh` - find the flakiest jobs this week (and the flakiest tests in each
-  job).
+* `flakes.sh` - find the flakiest jobs this week (and the flakiest tests in each job).
     - Uses `flakes.sql` to extract and group data from BigQuery
-    - Usage `flakes.sh | tee flakes-$(date %Y-%m-%d).json`
-    - Latest results: [flakes-latest](https://github.com/kubernetes/test-infra/blob/master/experiment/bigquery/flakes-latest.json)
+    - Usage: `flakes.sh | tee flakes-$(date %Y-%m-%d).json`
+    - Latest results: [flakes-latest.json](https://github.com/kubernetes/test-infra/blob/master/experiment/bigquery/flakes-latest.json)
```

Comment:
A [relative link](https://github.com/blog/1395-relative-links-in-markup-files) might be more appropriate here, instead of hardcoding the Github URL.